rust/src/main.rs

```diff
@@ -63,16 +63,16 @@ pub fn format_toml(content: &str, opt: &Settings) -> String {
     reorder_tables(&root_ast, &mut tables);
 
     let options = Options {
-        align_entries: false,           // do not align by =
-        align_comments: true,           // align inline comments
-        align_single_comments: true,    // align comments after entries
-        array_trailing_comma: true,     // ensure arrays finish with trailing comma
-        array_auto_expand: true,        // arrays go to multi line for easier diffs
-        array_auto_collapse: false,     // do not collapse for easier diffs
-        compact_arrays: false,          // do not compact for easier diffs
-        compact_inline_tables: false,   // do not compact for easier diffs
-        compact_entries: false,         // do not compact for easier diffs
-        column_width: opt.column_width, // always expand arrays per https://github.com/tamasfe/taplo/issues/390
+        align_entries: false,         // do not align by =
+        align_comments: true,         // align inline comments
+        align_single_comments: true,  // align comments after entries
+        array_trailing_comma: true,   // ensure arrays finish with trailing comma
+        array_auto_expand: true,      // arrays go to multi line when too long
+        array_auto_collapse: false,   // do not collapse for easier diffs
+        compact_arrays: false,        // leave whitespace
+        compact_inline_tables: false, // leave whitespace
+        compact_entries: false,       // leave whitespace
+        column_width: opt.column_width,
```

Comment:
I fixed the comments here, because several did not actually describe what the respective option does at all:

- `array_auto_expand: true` is conditional depending on `column_width`, and saying “for easier diffs” implies that it always happens, which is wrong
- `compact_*` has nothing to do with multiline vs single line, these options are about the whitespace within brackets, i.e. `["compact"]` vs `[ "not compact" ]`
- The comment for `column_width` was wrong in multiple ways:
  - “always expand arrays” would happen if you set it to `1`, which isn‘t done here
  - tamasfe/taplo#390 is a VS Code issue and therefore isn’t relevant at all